site/_docs/reference.md

```diff
@@ -2854,6 +2854,8 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b | UNIX_MILLIS(timestamp)                         | Returns the number of milliseconds since 1970-01-01 00:00:00
 | b | UNIX_SECONDS(timestamp)                        | Returns the number of seconds since 1970-01-01 00:00:00
 | b | UNIX_DATE(date)                                | Returns the number of days since 1970-01-01
+| s | URL_DECODE(string)                             | Decodes a *string* in 'application/x-www-form-urlencoded' format using a specific encoding scheme
```

Comment:
You should specify what happens if the function fails to encode/decode. Is the result NULL or a runtime error?

Reply:
Thanks for remind, I will check it soon.

Reply:
It will return original string when decoded error. I have fixed this and added the behaviour to doc.